codexctl/__init__.py

```
@@ -1,723 +1,418 @@
 
-        print("Success! Please restart the reMarkable device!")
+            except FileNotFoundError:
+                print(f"{args['target_path']}: No such file or directory")
+                raise FileNotFoundError
 
-    os.chdir(orig_cwd)
-    if temp_path:
-        logger.debug(f"Removing {temp_path}")
-        shutil.rmtree(temp_path)
+            except OSError:
+                print(f"{args['target_path']}: {os.strerror(e.errno)}")
```

Ruff (0.8.0)

Undefined name `e`

(F821)